earthpy/io.py

```diff
@@ -77,17 +76,37 @@ class EarthlabData(object):
     """
```

Comment:
I'd really like to rename this `EarthpyData()` to be consistent with the package name. I know that's not in scope for this PR but are you ok if i change it here or would you prefer a separate PR?

Reply:
I can change it in this PR - do you want `EarthpyData` over `EarthPyData`? We use the capitalization EarthPy in the documentation, and `EarthpyData` would deviate from how that convention. But, maybe `earthpy.io.EarthpyData` is redundant (two "earthpy"s)?

Another idea would be to call the class `Data`, so that we have `earthpy.io.Data`. What do you think @lwasser?

Reply:
i think the more generic it is, the better @mbjoseph  The only potential point of confusion with naming it data is that was instantiate it to `data` in the init file `data = EarthlabData()` so that would become `data = Data()`

so 
`data = Data()`
`data = EarthPyData()`
`data = AccessData()`

In theory i like doing it that way so on the front end the user doesn't have to work about instantiating an instance, they can just grab the data. Maybe then `data=Data()` is ok?

I think in general i don't expect users to call `earthpy.io.Data`  often. the most common use case will be `et.data.get_data()`

Reply:
Sounds good - I'll change it to `earthpy.io.Data` for now, and good point about this not being super important for users, who will still use the same old `et.data`.

Reply:
Changed in 9565786e7ffbcdec0507c17082a0f03a8cfcc4ed